spec/features/trainee_actions/filtering_trainees_spec.rb

```diff
@@ -46,8 +46,8 @@
     end
 
     scenario "can filter by status" do
```

Comment:
could we also add some tests to make sure the status chaining works? So chaining course_not_yet_started and in_training for example, maybe chain up to 3 or something?

Reply:
Added a status chain test, there was already a status chain unit test in but I have added one here for completeness